scripts/tools/metric.yml

```diff
@@ -115,10 +118,21 @@ etors:
       desc:
             "1.6": "Metric type: instruction pointer"
             "1.7": "Metric type: instruction pointer. Deprecated, use $T_METRIC_TYPE_IP."
-      value: "0x7ffffffe"
     - name: IP
       desc: "Metric type: instruction pointer"
       version: "1.7"
+    - name: EXP_EVENT_NO_VALUE
+      desc: "Metric type: event with only timestamp and value has no meaning"
```

Comment:
should it be "EVENT_EXP_TIMESTAMP" ?

Reply:
you mean final name ZET_METRIC_EVENT_EXP_TIMESTAMP ? Sure I'll update